### PR TITLE
Issue #516:hiding sensitive cmd info

### PR DESCRIPTION
**Pull request by shachiagarwal**
# Please include a summary of the changes

* vane/tests_tools.py - add hidden_cmd arg to run_show_cmds() func 
* sample_network_tests/vane_system_tests/test_vane.py - added a test_cmd_template() func to test hidden_cmd arg of run_show_cmds() func

# Any specific logic/part of code you need extra attention on
The case where cmd fails the exception generated by eapi/netmiko gets replaced with new message hiding the cmd run.

Explain any specific logic you need special attention on

# Include the Issue number and link

#516 

# List/Attach any dependencies/past issues that are required for this change/provide context

# Type of change

- - [ ] Bug fix (non-breaking change which fixes an issue)
- - [X] New feature (non-breaking change which adds functionality)
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- - [ ] This change requires a documentation update
- - [ ] Other (please specify)

# Effort required on reviewers end

- - [ ] Easy
- - [X] Medium
- - [ ] Hard 

# How Has This Been Tested?

## Bug fix

    Include before and after snapshots/screenshots/changed logs
    
## New feature

    Ensure current test cases pass and include newer test cases if required
    
# CI pipeline result

- - [X] Pass
- - [ ] Fail
  
  If it fails, explain the reason and whether or not we should ignore the failure
  
# Verify Documentation Update

    If applicable, ensure documentation has been updated for ReadMe, Getting Started guide, and Style guide
    
# Additional comments

### CODE REVIEW

File: vane/utils.py [hunk 1]
```diff
@@ -114,3 +115,20 @@ def now():
     """Return current date and time"""
 
     return (datetime.datetime.now()).strftime("%d-%m-%Y %H:%M:%S")
+
+
+def render_cmds(dut, cmds):
+    """
+    Method to convert cmds from jinja template using dut data
+
+    Args: dut - data used in template renedering
```
Comment (rewati-arista) [inline vane/utils.py:124]:
```suggestion
    Args: dut - data used in template rendering
```

File: vane/tests_tools.py [hunk 2]
```diff
@@ -1391,23 +1409,32 @@ def _run_and_record_cmds(
...
-                    json_results = conn.enable(cmds)
+                    json_results = conn.enable(run_cmds)
                 # also run the commands in text mode
-                txt_results = conn.enable(cmds, encoding="text")
+                txt_results = conn.enable(run_cmds, encoding="text")
             else:
                 # run the config cmd
                 txt_results = conn.config(cmds)
-        except BaseException as e:
+        except BaseException as e:  # pylint: disable=broad-except
```
Comment (rewati-arista) [inline vane/tests_tools.py:1425]: Can we add this disable at the top https://github.com/aristanetworks/vane/blob/36ceb86853bab1bc6fbe0e37ef767820796af323/vane/tests_tools.py#L31
I think when we reformatted files we started adding local pylint disables to the top of a file

File: sample_network_tests/vane_system_tests/test_vane.py [hunk 3]
```diff
@@ -529,3 +529,67 @@ def test_file_transfer_using_sftp(self, dut, tests_definitions):
...
+
+            # forming output message if test result is fail
+            if self.tops.expected_output != self.tops.actual_output:
+                self.tops.output_msg = (
+                    "SNMP traps are not generated and returned invalid output when snmpwalk command"
+                    " is executed locally on switch."
+                )
+
+        except (AssertionError, AttributeError, LookupError, EapiError) as excep:
+            logging.error(
```
Comment (rewati-arista) [inline sample_network_tests/vane_system_tests/test_vane.py:585]: We use f-string elsewhere in this file for logging

**Review by rewati-arista — APPROVED** [review]
LGTM! (some minor suggestions for styling)

File: vane/tests_tools.py (hunk 2 above)
Comment (shachiagarwal) [reply vane/tests_tools.py:1425]: Putting this at top of the file will disable it for entire file. I donot want that. I am just ignoring this warning here. Does it make sense?

**Review by kgrozis-arista — APPROVED** [review]

File: vane/tests_tools.py (hunk 2 above)
Comment (rewati-arista) [reply vane/tests_tools.py:1425]: Yep makes sense!

File: sample_network_tests/vane_system_tests/test_vane.py (hunk 3 above)
Comment (shachiagarwal) [reply sample_network_tests/vane_system_tests/test_vane.py:585]: https://github.com/aristanetworks/vane/issues/522 added this issue

**Pull request merged**